pkg/controllers/repo_config.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright © 2020 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package controllers
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/DopplerHQ/cli/pkg/models"
+	"github.com/DopplerHQ/cli/pkg/utils"
+	"gopkg.in/yaml.v3"
+)
+
+// repoConfigFileName (doppler.yaml)
+var repoConfigFileName string
+
+// RepoConfigFile (e.g. ./doppler.yaml)
+var RepoConfigFile string
+
+func init() {
+	repoConfigFileName = "doppler.yaml"
+	RepoConfigFile = filepath.Join("./", repoConfigFileName)
+}
+
+// RepoConfig Reads the configuration file (doppler.yaml) if exists and returns the set configuration
+func RepoConfig() models.RepoConfig {
+
+	if utils.Exists(RepoConfigFile) {
+		yamlFile, err := ioutil.ReadFile(RepoConfigFile) // #nosec G304
+
+		if err != nil {
+			utils.HandleError(err, "Unable to read doppler defaults file")
```

Comment:
Should we error here or just return an empty config? If we can't read anything for some reason, it might be better to just gracefully handle it and log a debug statement?

Reply:
Good question, I think we should do 3 things:

1. Log something for the user
2. Debug log the error object
3. Return an empty config

It might also be useful if `RepoConfig()` returns the error and allowed the caller to handle it. This would be similar to the `MetadataFile()`. I also think we should debug log when we're reading the file.